lux/core/frame.py

```diff
@@ -471,6 +472,7 @@ def maintain_recs(self):
                     lux.register_action("Enhance", enhance, one_current_vis)
                     lux.register_action("Filter", filter, one_current_vis)
                     lux.register_action("Generalize", generalize, one_current_vis)
+                    lux.register_action("Similarity", similar_pattern, one_current_vis, 15)
```

Comment:
Should we change the condition here to be `one_current_vis` and that one of the filter attributes are present, and that we have a line chart being displayed? Otherwise, similarity might not make a lot of sense without a filter attribute or a line chart.